cibuildwheel/linux.py

```diff
@@ -109,9 +109,13 @@ def build(project_dir, output_dir, test_command, test_requires, test_extras, bef
 
                         # Run the tests from a different directory
                         pushd $HOME
-                        sh -c {test_command}
+                        sh -o errexit -c {test_command}
```

Comment:
I'm maybe 50/50 on adding the `-o errexit` here, because `cmd1 && cmd2` is well recognised, and I'm not sure how to recreate `cmd1; cmd2` when `-o errexit` is enabled.

But if we're changing the behaviour of this, we should change before_build and the Mac and Windows versions too, for consistency.

Reply:
I think I agree with you and will remove the ``-o errexit`` settting from this command, using `` && `` is indeed well recognized.